src/diamond/handler/stats_d.py

```diff
@@ -152,7 +154,7 @@ def _connect(self):
             self.connection = statsd.StatsClient(
                 host=self.host,
                 port=self.port
-            )
+            ).pipeline()
```

Comment:
yup documented at http://statsd.readthedocs.org/en/v3.2.1/pipeline.html for https://pypi.python.org/pypi/statsd/ and introduced in version 2.0 according to jsocol/pystatsd@51d684e46d

afaict this collector supports both the above and https://pypi.python.org/pypi/python-statsd/1.7.2 with the former being actively maintained on github but not the latter, hence the conditional